tests/dbus/test_dbus_client.cpp

```diff
@@ -239,6 +242,34 @@ void CheckNat64(ThreadApiDBus *aApi)
 #endif
 }
 
+void CheckTelemetryData(ThreadApiDBus *aApi)
+{
+    OTBR_UNUSED_VARIABLE(aApi);
+#if OTBR_ENABLE_TELEMETRY_DATA_API
+    std::vector<uint8_t> responseTelemetryDataBytes;
+
+    TEST_ASSERT(aApi->GetTelemetryData(responseTelemetryDataBytes) == OTBR_ERROR_NONE);
+    // Print TelemetryData proto in hex format.
+    printf("TelemetryData bytes in hex: ");
+    for (uint8_t byte : responseTelemetryDataBytes)
+    {
+        printf("%02x ", byte);
+    }
+    printf("\n");
+    threadnetwork::TelemetryData telemetryData;
```

Comment:
Move up to get together with other variable definitions.

Reply:
Done.